Core/Entities/PointTripEntity/PointTrip.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Core.Entities.PointEntity;
+using Core.Entities.TripEntity;
+
+namespace Core.Entities.PointTripEntity
+{
+    public class PointTrip
```

Comment:
You can avoid the explicit definition of intermediate table because of auto-generated many to many relationships in the EF Core.